src/pages/billing/List.js

```diff
@@ -87,41 +255,64 @@ export const BillingList = ajaxCaller(class BillingList extends Component {
           }
         })
       ]),
-      h(PageBox, { style: { padding: '1.5rem', flex: 1 } }, [
+      h(PageBox, [
         div({ style: Style.cardList.toolbarContainer }, [
-          div({
-            style: {
-              ...Style.elements.sectionHeader,
-              textTransform: 'uppercase',
-              marginBottom: '1rem'
-            }
-          }, ['Billing Projects Management'])
+          div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Billing Projects Management']) //Billing Projects? Billing Management? Billing Project Management?
         ]),
         div({ style: Style.cardList.cardContainer }, [
+          h(NewBillingProjectCard, {
+            onClick: () => {
+              if (Auth.getAuthInstance().currentUser.get().hasGrantedScopes('https://www.googleapis.com/auth/cloud-billing')) {
+                this.setState({ creatingBillingProject: true })
+              } else {
+                const options = new window.gapi.auth2.SigninOptionsBuilder({ 'scope': 'https://www.googleapis.com/auth/cloud-billing' })
+                Auth.getAuthInstance().currentUser.get().grant(options).then(
+                  () => setTimeout(() => this.setState({ creatingBillingProject: true }), 250),
+                  () => reportError('Failed to grant permissions', 'To create a new billing project, you must allow Terra to view your Google billing account(s).')
+                )
+              }
+            }
+          }),
           div({ style: { flexGrow: 1 } }, [
             _.flow(
               _.filter(({ projectName }) => Utils.textMatch(filter, projectName)),
               _.map(project => {
                 return h(ProjectCard, {
                   project, key: `${project.projectName}`
-                  // onDelete: () => this.setState({ deletingProject: project })
                 })
               })
             )(billingProjects)
           ]),
           !isDataLoaded && spinnerOverlay
-        ])
-      ])
-
+        ]),
+        creatingBillingProject && h(NewBillingProjectModal, {
+          onDismiss: () => this.setState({ creatingBillingProject: false }),
+          onSuccess: () => this.refresh() // open the modal
```

Comment:
Do we need to close the modal here?

Reply:
The modal is closed by `this.refresh()` as it calls `this.setState({ creatingBillingProject: false })`.